### DISCUSSION TITLE
✔ A request: Could we get a (or a couple of) sample node(s) which implements all of the node options?

### DISCUSSION THREAD

**Discussion by WillAdams** [IndiePython/nodezator#9] [General]
That way, folks could open up the sample, edit it, then be pretty confident it would load/work.

Think of it as Literate Programming:

http://literateprogramming.com/

**Comment by KennedyRichard** [comment]
You can, in fact, those were the first nodes I ever produced, I have a lot of those. I just didn't release them from the beginning cause I thought it would be good to give people time to get used to the basics first, but I may be wrong about that. They might not have all options though, specially the most recent ones, but they already provide a lot of options, so they are a good start.

Give me some time to quickly review them and I should post the an example node pack later today.

**Comment by KennedyRichard** [comment]
The pack is already available! Here's the link: https://github.com/KennedyRichard/examples_node_pack

Don't forget to read the README file for extra instructions.

**Reply by WillAdams** [reply]
Thanks! That should be a big help.

**Comment by KennedyRichard** [comment]
Forgot to close the discussion. Doing it now!

Peace
